core/src/main/scala/cats/instances/vector.scala

```diff
@@ -73,6 +73,8 @@ trait VectorInstances extends cats.kernel.instances.VectorInstances {
 
       override def size[A](fa: Vector[A]): Long = fa.size.toLong
 
+      override def get[A](fa: Vector[A])(idx: Long): Option[A] = if (fa.size > idx && idx > 0) Some(fa(idx.toInt)) else None
```

Comment:
Ahh, I see you have the `Vector` here, but not yet List (which can be made faster than a full fold) or NonEmpties.